tests/dummy/app/pods/demo/template.hbs

```diff
@@ -1,9 +1,39 @@
-  <h1>frost-sidebar testbed</h1>
-
+<div class="demo-instance" >
```

Comment:
Did you double check all your files to make sure that the indentation is correct?